src/main/java/uk/gov/hmcts/reform/adoption/adoptioncase/model/CaseData.java

```diff
@@ -231,6 +235,20 @@
     )
     private List<ListValue<AdoptionDocument>> applicant1DocumentsUploaded;
 
+    @CCD(
+        label = "Documents uploaded",
+        typeOverride = Collection,
+        typeParameterOverride = "AdoptionDocument",
+        access = {DefaultAccess.class}
+    )
+    private List<ListValue<AdoptionDocument>> documentsUploaded;
```

Comment:
If applicable, please update label comments for documentsUpload, adoptionDocument to hold caseworker upload doc details

Reply:
Thanks @pankajhmcts , Any expert can upload document in xui, for now we have clarity only about caseworker role but there will be many roles who will be working on the case. So its name is fine but yes CCD label can be updated, once we have xui specific mockups provided.